frontend/components/panels/ProfitCheckPanel.vue

```diff
@@ -15,20 +19,30 @@ import BasePanel from '../common/BasePanel.vue';
 import TextBlock from '../common/TextBlock.vue';
 
 export default Vue.extend({
-    name: 'GrossProfitCheckPanel',
+    name: 'ProfitCheckPanel',
     components: { TextBlock, BasePanel },
     props: {
         grossProfit: {
             type: Object as Vue.PropType<BigNumber>,
             default: undefined,
         },
+        netProfit: {
+            type: Object as Vue.PropType<BigNumber>,
+            default: undefined,
+        },
     },
     computed: {
         currentStateAndTitle(): PanelProps {
-            if (this.grossProfit === undefined) {
+            if (this.grossProfit === undefined || this.netProfit === undefined) {
                 return {
-                    name: 'notice',
-                    title: `The transaction gross profit is unknown`,
+                    name: 'incorrect',
+                    title: `The transaction profit is unknown`,
+                };
+            }
```

Comment:
Since we have two different profits, let's always specify which one are we talking about. Here and everywhere else:
```suggestion
            if (this.grossProfit === undefined || this.grossProfit?.isNaN()) {
                return {
                    name: 'incorrect',
                    title: `The transaction gross profit is unknown`,
                };
            }
```